caravel/viz.py

```diff
@@ -351,10 +351,13 @@ def get_df(self, query_obj=None):
 
     def get_data(self):
         df = self.get_df()
-        return dict(
+        dict_obj = dict(
             records=df.to_dict(orient="records"),
-            columns=list(df.columns),
+            columns=list(df.columns)
         )
+        # convert Timestamp in the dict_obj to iso timestamp by default.
+        json_conversion = json.dumps(dict_obj, default=utils.json_iso_dttm_ser,)
```

Comment:
Hmm... I took this logic from panoramix 0.7 but in the current version I get unix timestamps :/ 
One way would be to do the conversions while rendering data after the payload is created. 
Any suggestions what's the best way to handle this? I can then push the right fix